plugin/core/configurations.py

```diff
@@ -75,10 +34,14 @@ def apply_project_overrides(client_config: ClientConfig, lsp_project_settings: d
     return client_config
 
 
-def is_supported_syntax(syntax: str, configs: List[ClientConfig]) -> bool:
-    for config in configs:
-        if config_supports_syntax(config, syntax):
-            return True
+def is_supported_syntax(syntax: str, configs: Iterable[ClientConfig]) -> bool:
+    try:
+        scope = syntax2scope(syntax)
+        for config in configs:
+            if config.supports(scope):
+                return True
+    except StopIteration:
+        debug(syntax, "is not in sublime.list_syntaxes()")
```

Comment:
I'm getting "million" of those:
```
LSP: Packages/Default/Find Results.hidden-tmLanguage is not in sublime.list_syntaxes()
```
Is it really useful to print those?
If you think so then I guess it might work to exclude the `hidden-*` ones.

Reply:
I've removed the message.